opensrp-giz-malawi/src/main/res/layout/fragment_me.xml

```diff
@@ -0,0 +1,386 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
We don't need this file, nothing is overriden. Can you countercheck? And get back in case there is anything new. Comparing both files https://github.com/OpenSRP/opensrp-client-anc/blob/master/opensrp-anc/src/main/res/layout/fragment_me.xml & this one does not show any difference